.github/workflows/tests.yml

```
@@ -91,4 +91,5 @@ jobs:
     uses: cloudposse/.github/.github/workflows/shared-go-auto-release.yml@main
     with:
       publish: true
+      runs-on: '["runs-on=${{ github.run_id }}", "runner=default", "extras=s3-cache"]'
```

We cannot use the `s3-cache` until all jobs run on the runner start with `uses: runs-on/action@v1`

```suggestion
      runs-on: '["runs-on=${{ github.run_id }}", "runner=default"]'
```